src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/custom/DSLParamStrategy.java

```diff
@@ -24,15 +24,18 @@ public String toDSL() {
 	public String getOrderedChildrenDSL() {
 		PropertyDescriptor propertyDescriptor = (PropertyDescriptor) getDescriptor();
 		if (propertyDescriptor.getName().equals("hudson.model.StringParameterDefinition") || propertyDescriptor.getName().equals("hudson.model.BooleanParameterDefinition")) {
-			String defaultValue = "\"\"";
+			String defaultValue = "\"\"", description = "\"\"";
 			List <PropertyDescriptor> children = propertyDescriptor.getProperties();
 			for (PropertyDescriptor child : children ) {
 				if (child.getName().equals("defaultValue")) {
 					defaultValue = getChildrenByName("defaultValue").toDSL();
+
+				}
+				if (child.getName().equals("description")) {
+					description = getChildrenByName("description").toDSL();
```

Comment:
In the xml, the description or defaultValue tags may be missing entirely, which causes an exception when calling the getChildrenByName function and the job fails to translate. Added default empty strings for those values and only change them if the tag exists .